src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -141,20 +150,26 @@ export interface ConversationProps {
   refetch: RelayRefetchProp["refetch"]
 }
 
+export const PAGE_SIZE: number = 30
```

Comment:
**thought (non-blocking):** It might make sense to lower this page size being that we're paginating now. Loading 30 messages up front might be more than can display on the screen initially. (we'd need to verify that). 

<!--
Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.
-->

Reply:
Fair enough. Any preference?
As to how many can fit initially, I think might come down to screen size 🤔

Reply:
Just eyeballing it, on desktop for me there's room for ~15 entries. Maybe lowering it to 20? It'd be significantly less than that on mobile I presume, so that's a decent tradeoff.